.github/workflows/release-rancher-desktop.yml

```diff
@@ -27,7 +28,7 @@ jobs:
       run: |
         mkdir ${{ env.RELEASE_DIR }}
         yarn install --frozen-lockfile
-        NUXT_ENV_commit=${GITHUB_SHA} NUXT_ENV_version=${GITHUB_REF_NAME} OUTPUT_DIR=${{ env.OUTPUT_DIR }}/${{ env.ARTIFACT_NAME }} ROUTER_BASE='/dashboard' RANCHER_ENV=desktop yarn run build --spa
+        NUXT_ENV_commit=${GITHUB_SHA} NUXT_ENV_version=${GITHUB_REF_NAME} OUTPUT_DIR=${{ env.OUTPUT_DIR }}/${{ env.ARTIFACT_NAME }} API=${{ env.STEVE_API }} RESOURCE_BASE=${{ env.STEVE_API }} ROUTER_BASE='/dashboard' RANCHER_ENV=desktop yarn run build --spa
```

Comment:
Is `RESOURCE_BASE=${{ env.STEVE_API }}` correct?

Reply:
Yes, because the Steve is serving both our SPA and API when running a local Steve instance